packages/katello/katello/katello-service

```diff
@@ -1,98 +1,7 @@
-#!/usr/bin/env ruby
+#!/usr/bin/bash
```

Comment:
bash is usually found in `/bin`. Might be better to just depend on `/bin/sh` given the simplicity of the script.